src/proto/Filecoin.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package TW.Filecoin.Proto;
+option java_package = "wallet.core.jni.proto";
+
+// Input data necessary to create a signed transaction.
+message SigningInput {
+    // Private key of sender account.
+    bytes private_key = 1;
+
+    // Recipient's address.
+    string to_address = 2;
+
+    // Transaction nonce.
+    uint64 nonce = 3;
+
+    // Transfer value (256-bit number).
+    bytes value = 4;
```

Comment:
since decimals is `8`, we can use `int64`, same for gas price and gas limit - this should not cause overflow

Reply:
Okay, changed it to `uint64`. Hopefully this won't cause any troubles.